pyrdp/parser/rdp/fastpath.py

```diff
@@ -409,33 +400,19 @@ def writeBitmapEvent(self, stream: BytesIO, event: FastPathBitmapEvent):
         stream.write(event.payload)
 
     def parseOrdersEvent(self, stream: BytesIO, header: int, compressionFlags: int, size: int) -> FastPathOrdersEvent:
-        orderCount = Uint16LE.unpack(stream)
-        orderData = stream.read(size - 2)
-
-        assert len(orderData) == size - 2
+        """
+        Parse the order events from a TS_FP_UPDATE_ORDERS.
+        This is specified in MS-RDPEGDI.
+        """
+        payload = stream.read(size)
+        assert len(payload) == size
```

Comment:
I don't remember if we use `assert` in PyRDP, but putting an error message as a second argument would help for the end user to understand exactly what went wrong

Reply:
Asserts are not meant to be user-consumable errors. If this condition is false, the stream is either malformed or corrupted. I will be adding error handling code that skips unknown PDUs and optionally logs them to serve as unit tests.

Reply:
I've added the code I was talking about in db459c9